pyiron_atomistics/lammps/base.py

```diff
@@ -371,19 +372,21 @@ def enable_h5md(self):
             "all h5md ${dumptime} dump.h5 position force create_group yes"
         )
 
-    def write_input(self):
+    def get_input_file_dict(self) -> dict:
         """
-        Call routines that generate the code specific input files
+        Get an hierarchical dictionary of input files. On the first level the dictionary is divided in file_to_create
+        and files_to_copy. Both are dictionaries use the file names as keys. In file_to_create the values are strings
+        which represent the content which is going to be written to the corresponding file. In files_to_copy the values
+        are the paths to the source files to be copied.
```

Comment:
```suggestion
        are the paths to the source files to be copied.

        The get_input_file_dict() function is called before the write_input() function to convert the input specified on
        the job object to strings which can be written to the working directory as well as files which are copied to the
        working directory. After the write_input() function wrote the input files the executable is called. 
```

Reply:
@samwaseda Like so?

Reply:
Yep. It didn't have to be so detailed but it looks good to me.